shiny/examples/input_file/app.py

```diff
@@ -0,0 +1,48 @@
+import pandas as pd
+
+from shiny import App, reactive, render, ui
+
+app_ui = ui.page_fluid(
+    ui.input_file("file", "File", accept=".csv"),
+    ui.input_checkbox_group(
+        "stats", "Summary Stats", choices=["Row Count", "Column Count", "Column Names"]
+    ),
+    ui.output_table("summary"),
+)
+
+
+def server(input, output, session):
+    @reactive.Calc
+    def parsed_file():
+        file = input.file()
```

Comment:
Similar to above, having the type annotation can be helpful:

```suggestion
        file: list[FileInfo] = input.file1()
```